components/dashboard/src/teams/git-integrations/GitIntegrationsList.tsx

```diff
@@ -43,7 +43,7 @@ export const GitIntegrationsList: FunctionComponent<Props> = ({ providers }) =>
                 <EmptyMessage
                     title="No Git Auth configurations"
                     subtitle="Configure Git Auth with GitHub or GitLab."
-                    buttonText="New Integration"
+                    buttonText="New Git Auth"
```

Comment:
**issue(non-blocking)**: Button labels like this can be confusing to users, especially when they are shortened. In any case, this should be blocking for this PR.

Reply:
what do you suggest we make it instead? I was changing copy away from "Git Integrations" to "Git Auth" - and this was a leftover still referencing Integration.

Reply:
@selfcontained I'd go with _Git Integrations_ or _Git Connections_ but let's update this once there's consensus around the change, see [relevant discussion](https://gitpod.slack.com/archives/C02EN94AEPL/p1679923278367079) (internal). Cc @AlexTugarev @geropl @easyCZ @atduarte

Reply:
I think that confusion is my fault, as I gave such a feed on a prev PR. Actually, I mixed up the views, indeed from Org admin perspective you are _integrating_ with a Git provider, and users may then _authenticate_ with that Git provider to access their repositories.

### Org Settings

<img width="563" alt="Screenshot 2023-03-28 at 07 56 38" src="https://user-images.githubusercontent.com/914497/228143037-739d5ba3-7bad-418c-901f-3deda7d41fa5.png">

### User Settings

<img width="720" alt="Screenshot 2023-03-28 at 08 00 18" src="https://user-images.githubusercontent.com/914497/228143047-707215bb-51d8-4493-883d-ee954b7b93c7.png">